### PR TITLE
services: Change group of avahi services to avahi-service-writers

### PR DESCRIPTION
**Pull request by smspillaz**
This will allow users in that group to add and remove new services persistently.

Also removed the old companion app services, as they will be relocated to eos-companion-app-integration as part of this ticket.

https://phabricator.endlessm.com/T20391

### CODE REVIEW

**Comment by smspillaz** [comment]
Not sure why jenkins failed, the relevant file `eos-companion-app.service` isn't referred to anywhere in the makefile..

File: Makefile.am [hunk 1]
```diff
@@ -12,6 +12,7 @@ AM_DISTCHECK_CONFIGURE_FLAGS = \
 	$(NULL)
 
 dist_systemdunit_DATA = \
+	eos-chown-avahi-helper-to-writers.service \
 	eos-companion-app.service \
 	eos-companion-app.socket \
```
Comment (cosimoc) [inline Makefile.am]: The files you remove are referenced here

Comment (smspillaz) [reply Makefile.am]: Agh, forgot to add my new changes 😅

**Comment by smspillaz** [comment]
Closing in favour of a different approach

**Comment by pwithnall** [comment]
> Closing in favour of a different approach

(See https://phabricator.endlessm.com/T20391#523023)

**Pull request closed without merging**